app/views/daac_curator_mailer/released_records_digest_notification.html.erb

```diff
@@ -13,7 +13,7 @@
     <tbody>
       <% @records.first(50).each do |record| %>
         <tr>
-          <td><%= link_to(record.collection? ? record.short_name : record.long_name, reports_review_url(record_id: record.id)) %></td>
+          <td><%= link_to(record.short_name, reports_review_url(record_id: record.id)) %></td>
```

Comment:
Don't need this logic to accommodate granules since we will not be displaying them anymore